templates/about.html

```diff
@@ -12,7 +12,7 @@ <h2 class="m-0 font-bold text-lg">About this job board</h2>
   </p>
   <p>
     Job postings for non-regional employers that also do not have any local employees are fine <strong>as long as they
-      are posted by local commnuity members</strong>.
+      are posted by local community members</strong>.
   </p>
   <p>
     The goal is to avoid individuals / companies with no tie to the region from dropping random listings here. There are
```

Comment:
Could you grab this one while here?
```suggestion
    The goal is to avoid individuals or companies with no tie to the region from dropping random listings here. There are
```